.github/workflows/release-project-sync.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow runs release_project_sync.py on demand
+
+name: Fix Transifex resource names and slugs
+
+on:
+  workflow_dispatch:  # by request
+
+jobs:
+  release-project-sync:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Clones the openedx-translations repo
+      - name: clone openedx/openedx-translations
+        uses: actions/checkout@v3
+
+      # Sets up Python
+      - name: setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+
+      # Run the script
+      - name: Sync reviews for release project
+        env:
+          TRANSIFEX_API_TOKEN: ${{ secrets.TRANSIFEX_API_TOKEN }}
+        run: |
+          make translations_scripts_requirements
+          python ./scripts/release_project_sync.py redwood
```

Comment:
another non-blocking comment - we'll likely want to update this to not be hardcoded and instead be an option like we have in https://github.com/openedx/openedx-translations/blob/main/.github/workflows/extract-translation-source-files.yml

Reply:
done.

> Only comment I have is that some of the code is quite similar to code in https://github.com/openedx/openedx-translations/blob/main/scripts/fix_transifex_resource_names.py, so it might be nice to pull some of that out into a shared file at some point, but that absolutely should not block merging/running this!

I think the reuse might be overrated here since it's mostly a single method for getting the project. But a noteworthy point for sure.